xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGL.cpp

```diff
@@ -303,7 +303,7 @@ bool CLinuxRendererGL::Configure(unsigned int width, unsigned int height, unsign
   // on osx 10.9 mavericks we get a strange ripple
   // effect when rendering with pbo
   // when used on intel gpu - we have to quirk it here
-  if (CDarwinUtils::IsMavericks())
+  if (CDarwinUtils::IsMavericksOrHigher())
```

Comment:
@HofiOne sorry for confusion, see my seconds post. By giving this method the correct name, you brought an already existing issue to surface. This is good! :)